apps/portals/src/configurations/genie/logoHeaderConfig.ts

```diff
@@ -0,0 +1,6 @@
+import { LogoConfig } from 'types/portal-config'
+import icon from './style/GENIE-logo.jpeg'
```

Comment:
Heads up, since #808 was merged, this will have to change:
```suggestion
import icon from './style/GENIE-logo.jpeg?url'
```

Reply:
This JPEG import probably does not need to be updated, it works both ways. I think the '?url' style is clearer, though.